src/EnergyPlus/EPVector.hh

```diff
@@ -72,6 +74,18 @@ template <typename T> struct EPVector : private std::vector<T>
     using value_type = T;
     using size_type = typename std::vector<T>::size_type;
 
+#ifndef NDEBUG
+    [[nodiscard]] T &operator[](std::size_t n)
+    {
+        return std::vector<T>::at(n);
```

Comment:
For debug builds, use the base `at` member function, which throws for bad indices.